packages/core/src/server/service.ts

```diff
@@ -28,10 +28,13 @@ export class ServerService {
     this.port = this.common.options.port;
   }
 
+  #getOwnUrlFromRequest(request: express.Request) {
+    return `${request.protocol}://${request.get("host")}`;
+  }
```

Comment:
This is a style thing, but TBH I'd probably inline this in both places rather than create a method.

Reply:
done